### ISSUE TITLE
Stat summary doesn't show up on russian client

### ISSUE THREAD

**Issue by thehallowedfire** [raethkcj/RatingBuster#91]
Hello.
I have 'Show stat summary' turned on but it doesn't show up. On english client it does show up, on russian - doesn't.
![image](https://user-images.githubusercontent.com/112851853/189461663-bc938ba3-03ca-4690-bfe0-0ac64d97c2ad.png)

**Comment by thehallowedfire** [comment]
Also is it possible to move calculated numbers (that's inside parenthesis) to the end of the stroke? For example:
from
+25 (+0.35% crit, +4.2 dmg) Intellect
to
+25 Intellect (+0.35% crit, +4.2 dmg)

Can't recall how exactly it looks on english client, but on russian it works like this (ex. 1) and creates a little bit of mess.

**Comment by raethkcj** [comment]
Hi, apparently I wasn't loading `libs/StatLogic/locales/ruRU.lua`, so all of the summary localization was missing. I added that in the latest release, so a few stats should show up now. However, a lot of the translations are still missing in that file, as mentioned in https://github.com/raethkcj/RatingBuster/issues/9#issuecomment-851055849. I've tried to start those translations a few times but I just can't tell the Cyrillic characters apart and I don't make much progress.

As for the location of the numbers in parenthesis, those were intended to be at the end like you said, I've fixed that as well for version 1.5.4.
